create-custom-map-style/README.md

```diff
@@ -0,0 +1,990 @@
+	import  awsExports  from  './aws-exports';
+
+	Amplify.configure(awsExports);
+	
+	ReactDOM.render(
+		<React.StrictMode>
+		  <App  />
+		</React.StrictMode>,
+		document.getElementById('root')
+	);
+
+
+	// If you want to start measuring performance in your app, pass a function
+	// to log results (for example: reportWebVitals(console.log))
+	// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
+	reportWebVitals();
+	```
+	
+6. After configuring Amplify at the React application's top-level module it can be used in child components. In our particular case this is only `App.js`
+
+7. Before we proceed we will need to bring in a few more dependencies, in your terminal run the following commands:
+	```
+	npm i @aws-amplify/ui-react
+	```
+
+8. Open `src/App.js` in your IDE. Under your existing list of existing import statements create a new line and add the following:
+	```
+	...
+	import { withAuthenticator } from  '@aws-amplify/ui-react';
+	```
+9. Replace your module export line - `export default App;` - with the following:
+	```
+	export default withAuthenticator(App);
+	``` 
+
+10. Amplify's React library provides an extremely useful higher order component to provide authentication for your app. By simply wrapping your application's existing code in this one component you now have a complete authentication flow (login/logout)
+
+11. To test out your app's new login screen open your terminal and start the react app: 
+	```
+	npm start
+	```
+	You should see a login screen as shown below. Being that this is the first time you are running the app with authentication you will need to create a new account to login
+
+	![Cognito Hosted UI Login Screen](media/Login.png)
+	*The Cognito Hosted UI Login Screen*
+
+12. Once you've created a new account and logged in you should be presented with the default React app UI
+
+	![Create React App Screen](media/CRA.png)
+	*The create react app default screen*
+
+### Creating a Map UI view
+
+ 1. To create this view we'll start with the HTML / CSS skeleton before adding the business logic with JS. I've renamed my `App` component (`src/App.js` and `src/App.css`) to `SamplePage`, but you can feel free to keep the name `App.js`. Open this file in your IDE.
+ 
+2. Inside the `return` statement we're going to erase everything and start from scratch. At the root we're going to start with a simple `div` element with a `class` attribute of `sample-page`. Your functional component should look something like this:
+	```
+	function SamplePage() {
+	  return (
+	    <div className="sample-page">
+	     
+	    </div>
+	  );
+	}
+	```
+3. Next we'll add in some sample content, a level one header with the title of the page:
+	```
+	<div className="sample-page">
+		<h1>Amazon Location Service - Create Your Own Custom Map Style</h1>
+	</div>
+	```
+
+4. Then we'll add a container for the kind of map we're going to demo
+	```
+	// Below h1
+	...
+	  <div className="maputnik-container">
```

Comment:
Maybe just `map-container`? I'm confused what this has to do with Maputnik.

Reply:
renamed it to `map-container`